### ISSUE TITLE
"I'm sorry, but an uncaught exception occurred."

### ISSUE THREAD

**Issue by Acituanbus** [Skull217/DDLCtVN#22]
So I unzipped the folder, put all the files into my /game folder and this comes up whenever I launch the game. I tried this on a fresh installation and am pretty sure I did everything correct :/

While running game code:
ScriptError: Name (u'C:\\Users\\Skull\\Desktop\\All\\Not Games\\DDLC Project\\project dir\\DDLCtVN/game/mod_assets/MonikaSittingExpressions/monika_sitting_expressions.rpy', 1518666830, 0) is defined twice, at game/mod_assets/MonikaSittingExpressions/monika_sitting_expressions.rpy:5 and game/mod_assets/MonikaSittingExpressions/monika_sitting_expressions.rpy:5.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "I:\Steam\steamapps\common\Doki Doki Literature Club\renpy\bootstrap.py", line 295, in bootstrap
    renpy.main.main()
  File "I:\Steam\steamapps\common\Doki Doki Literature Club\renpy\main.py", line 364, in main
    renpy.game.script.load_script()  # sets renpy.game.script.
  File "I:\Steam\steamapps\common\Doki Doki Literature Club\renpy\script.py", line 265, in load_script
    self.load_appropriate_file(".rpyc", ".rpy", dir, fn, initcode)
  File "I:\Steam\steamapps\common\Doki Doki Literature Club\renpy\script.py", line 753, in load_appropriate_file
    self.finish_load(stmts, initcode, filename=fn + source)
  File "I:\Steam\steamapps\common\Doki Doki Literature Club\renpy\script.py", line 419, in finish_load
    bad_node.filename, bad_node.linenumber))
ScriptError: Name (u'C:\\Users\\Skull\\Desktop\\All\\Not Games\\DDLC Project\\project dir\\DDLCtVN/game/mod_assets/MonikaSittingExpressions/monika_sitting_expressions.rpy', 1518666830, 0) is defined twice, at game/mod_assets/MonikaSittingExpressions/monika_sitting_expressions.rpy:5 and game/mod_assets/MonikaSittingExpressions/monika_sitting_expressions.rpy:5.

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187

Here's a picture of my /game folder: 
https://imgur.com/a/5IgXv3C

**Comment by Zanther123** [comment]
Yeah i figured out your problem you put the original files in the game folder which is good but you pulled out the mod assets folder files and put them with the original files. what you need to do is just click and drag all the files from the mod folder into the game folder but what you do not go into any other folders all you do is just go into the DDLCtVN-master folder and thats it dont touch anything else in the DDLCtVN-master folder all you do is open it and dont do anything else. then go to the game folder and just highlight all the files of the the DDLCtVN-master folder into the game folder that's it nothing else nothing more.

**Comment by Acituanbus** [comment]
Alright, got a different error now :/
Here's the error and my folder:
https://imgur.com/a/2GAeTBh

**Comment by Acituanbus** [comment]
Nevermind, I fixed it by deleting script2.rpy, but now a few minutes into playing this happens:
https://imgur.com/a/AMvKeeQ

**Comment by TurtleOverlordIV** [comment]
Welp, this is fun. 

Full traceback:
  File "lib/windows-i686/mods/script-poemgame.rpyc", line 1, in script
  File "C:\Users\anon\Downloads\ddlc-win\DDLC-1.1.1-pc\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\anon\Downloads\ddlc-win\DDLC-1.1.1-pc\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/mods/script-poemgame.rpy", line 20, in <module>
    with renpy.file('mod_assets/poem-game_words.txt') as wordfile:
  File "C:\Users\anon\Downloads\ddlc-win\DDLC-1.1.1-pc\renpy\exports.py", line 1980, in file
    return renpy.loader.load(fn)
  File "C:\Users\anon\Downloads\ddlc-win\DDLC-1.1.1-pc\renpy\loader.py", line 543, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'mod_assets/poem-game_words.txt'.

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187

**Comment by MrGus47** [comment]
My problem is

I'm sorry, but an uncaught exception occurred.

While running game code:
   File "game / script-poemgame.rpy", line 349, in script
     return
Exception: Could not find return label (u'D: \\ Documents \\ DDLC \\ DDLC / game / script.rpy ', 1475534846, 132).

- Full Traceback ---------------------------------------------- --------------

Full traceback:
   File "game / script-poemgame.rpy", line 349, in script
     return
   File "C: \ Users \ LuizGustavo \ Desktop \ DDLC-1.1.0-pc \ renpy \ ast.py", line 1376, in execute
     next_node (renpy.game.context (). lookup_return (pop = True))
   File "C: \ Users \ LuizGustavo \ Desktop \ DDLC-1.1.0-pc \ renpy \ execution.py", line 552, in lookup_return
     raise Exception ("Could not find return label {! r}.") (self.return_stack [-1]))
Exception: Could not find return label (u'D: \\ Documents \\ DDLC \\ DDLC / game / script.rpy ', 1475534846, 132).

Windows-7-6.1.7601-SP1
Ren'Py 6.99.12.4.2187
Doki Doki, The Normal VN! 1.4.9

Does anybody know how to solve this? please help me.

**Comment by miluxo** [comment]
I don't know if they are still helping but I have the same problem.

I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/chess.rpy", line 394, in script
  File "game/chess.rpy", line 456, in <module>
NameError: name 'mas_games' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\luisg\OneDrive\Escritorio\ddlce\DDLC-1.1.1-pc - just monika\renpy\bootstrap.py", line 295, in bootstrap
    renpy.main.main()
  File "C:\Users\luisg\OneDrive\Escritorio\ddlce\DDLC-1.1.1-pc - just monika\renpy\main.py", line 419, in main
    game.context().run(node)
  File "game/chess.rpy", line 394, in script
  File "C:\Users\luisg\OneDrive\Escritorio\ddlce\DDLC-1.1.1-pc - just monika\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\luisg\OneDrive\Escritorio\ddlce\DDLC-1.1.1-pc - just monika\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/chess.rpy", line 456, in <module>
NameError: name 'mas_games' is not defined

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187

**Comment by ggg007-beep** [comment]
[code]
I'm sorry, but an uncaught exception occurred.

While running game code:
ScriptError: could not find label 'day18_Back_home_kitchen'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "script4.rpyc", line 3182, in script
  File "C:\Users\HP\Downloads\A_Wife_And_Mother-0.100-pc\AWAM-0.100-pc\1 GAME FOLDER\renpy\ast.py", line 1697, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\HP\Downloads\A_Wife_And_Mother-0.100-pc\AWAM-0.100-pc\1 GAME FOLDER\renpy\script.py", line 898, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'day18_Back_home_kitchen'.

Windows-8-6.2.9200
Ren'Py 7.3.5.606
A Wife And Mother Vers. 0.100 0.100
Sun Sep 27 10:37:49 2020
[/code]

this is my issue is there a fix for this ??

**Comment by Zanther123** [comment]
Sorry to say but i haven't solved doki doki problems in a long time but my
suggestion used to be to replace all the files with the newest update or to
first erase all the files and try to install them again but besides that I
can't help you any further. Gl with your problem friend!

**Comment by DeDunot** [comment]
anyone know how to solve this? Would really appreciate it.


I'm sorry, but an uncaught exception occurred.

Before loading the script.
error: Error -3 while decompressing data: unknown compression method

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/Users/RudSars/Desktop/BeingADIK.app/Contents/Resources/autorun/renpy/bootstrap.py", line 314, in bootstrap
    renpy.main.main()
  File "/Users/RudSars/Desktop/BeingADIK.app/Contents/Resources/autorun/renpy/main.py", line 308, in main
    renpy.loader.index_archives()
  File "/Users/RudSars/Desktop/BeingADIK.app/Contents/Resources/autorun/renpy/loader.py", line 136, in index_archives
    index = loads(f.read().decode("zlib"))
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/encodings/zlib_codec.py", line 43, in zlib_decode
error: Error -3 while decompressing data: unknown compression method

Darwin-19.4.0-x86_64-i386-64bit
Ren'Py 7.3.2.320
 
Wed Apr 28 11:57:28 2021
